tests/fixtures/addon/with-concat-files/index.js

```diff
@@ -0,0 +1,19 @@
+/* jshint node: true */
+'use strict';
+
+var path = require('path');
+
+module.exports = {
+  name: 'concat-some-files',
+
+  included: function(app) {
+    this._super.included(app);
+
+    var vendor = this.treeGenerator(path.join(__dirname, 'vendor'));
+
+    return this.concatFiles(vendor, {
```

Comment:
Either way though, the method was completely untested (it would have errored loudly upon any invocation). I'd be in favor of deprecating this publicly in beta.3...